.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,33 +1,26 @@
 ---
 name: Bug report
 about: Filling a bug report
-title: ''
+title: ""
 labels: bug
-assignees: ''
-
+assignees: ""
 ---
 
-## Common project specific issues
-
-There are certain project specific issues that are commonly misinterpreted as bugs.
-
-- [Offerings, products, or available packages are empty](https://support.revenuecat.com/hc/en-us/articles/360041793174)
-- [Invalid Play Store credentials errors](https://support.revenuecat.com/hc/en-us/articles/360046398913)
-- [Unable to connect to the App Store (STORE_PROBLEM) errors](https://support.revenuecat.com/hc/en-us/articles/360046399333)
-
-For support I'd recommend our [online community](https://spectrum.chat/revenuecat), [StackOverflow](https://stackoverflow.com/tags/revenuecat/) and/or [Help Center](https://support.revenuecat.com/hc/en-us) 👍
-
-If you have a clearly defined bug (with a [Minimal, Complete, and Reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)) that is not specific to your project, follow the steps below to file it with RevenueCat. For SDK-related bugs, make sure they can be reproduced on a physical device, not a simulator (there are simulator-specific problems that prevent purchases from working).
+- [ ] I have updated Purchases SDK to the latest version
+- [ ] I have read the [Contribution Guidelines](https://github.com/RevenueCat/cordova-plugin-purchases/blob/develop/CONTRIBUTING.md)
+- [ ] I have read the [Help Center](https://support.revenuecat.com/hc/en-us)
+- [ ] I have read [docs.revenuecat.com](https://docs.revenuecat.com/)
+- [ ] I have searched for [existing Github issues](https://github.com/RevenueCat/cordova-plugin-purchases/issues)
 
 **Describe the bug**
-A clear and concise description of what the bug is. The more detail you can provide the faster our team will be able to triage and resolve the issue.
+A clear and concise description of what the bug is. The more detail you can provide the faster our team will be able to triage and resolve the issue. If debug logs are not applicable to this issue, please do not remove step 2 when submitting the issue.
 
 1. Environment
-    1. Platform:
-    2. SDK version:
-    3. OS version:
-    4. XCode/Android Studio version:
-    5. How widespread is the issue. Percentage of devices affected.
+   1. Platform:
+   2. SDK version:
+   3. OS version:
+   4. XCode/Android Studio version:
+   5. How widespread is the issue. Percentage of devices affected.
```

Comment:
I would ask if they are using Cordova, Ionic, Ionic Native, etc. and their versions. Also the OS where they develop.

Reply:
Just added!